lib/fileMapper.ts

```diff
@@ -197,7 +197,7 @@ async function fetchAndWriteFileStream(
     logger.log(i18n(`${i18nKey}.skippedExisting`, { filepath }));
     return;
   }
-  if (!isAllowedExtension(srcPath)) {
+  if (!isAllowedExtension(srcPath, ['tsx', 'ts', 'jsx'])) {
```

Comment:
This feeds back to this set of constants -- https://github.com/HubSpot/hubspot-local-dev-lib/blob/main/constants/extensions.ts#L1 - I see the list is being used for the watch/uploadFile area as well, so I can see the short fix here since the file types aren't 100% supported around the board. Perhaps we can make another set there with the CMS React extensions to that we can continue to use them around the lib as any further work progresses -- then we don't have to peel this out down the line or remember where this is set.

Reply:
Threw it into a const Set and pulled from there instead 👍